templates/eks-values.yaml

```diff
@@ -0,0 +1,8 @@
+api:
```

Comment:
Are these values files needed? I think we can just delete them

Reply:
they serve as example on how to set the values, so people have the template for each distro to follow